llvm/include/llvm/BinaryFormat/ELFRelocs/ARM.def

```diff
@@ -143,3 +143,10 @@ ELF_RELOC(R_ARM_THM_BF16,               0x88)
 ELF_RELOC(R_ARM_THM_BF12,               0x89)
 ELF_RELOC(R_ARM_THM_BF18,               0x8a)
 ELF_RELOC(R_ARM_IRELATIVE,              0xa0)
+ELF_RELOC(R_ARM_GOTFUNCDESC,            0xa1)
+ELF_RELOC(R_ARM_GOTOFFFUNCDESC,         0xa2)
+ELF_RELOC(R_ARM_FUNCDESC,               0xa3)
+ELF_RELOC(R_ARM_FUNCDESC_VALUE,         0xa4)
+ELF_RELOC(R_ARM_TLS_GD32_FDPIC,         0xa5)
+ELF_RELOC(R_ARM_TLS_LDM32_FDPIC,        0xa6)
+ELF_RELOC(R_ARM_TLS_IE32_FDPIC,         0xa7)
```

Comment:
I might be looking in the wrong place, but I couldn't see the values for these three relocations defined in the doc you link in the description?

Reply:
Thanks for checking. https://github.com/mickael-guene/fdpic_doc doesn't define these values, but binutils-gdb defines the values (also referenced by projects like https://github.com/cutty/uld-fdpic/blob/master/include/elf.h). I've checked that `llvm-readelf -r` output matches `readelf -r`.